src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/ProfileSetup/NIPHLNumberStepDef.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.test.ui.cucumber.stepdefs.ProfileSetup
+
+import uk.gov.hmrc.test.ui.cucumber.stepdefs.BaseStepDef
+import uk.gov.hmrc.test.ui.pages.AuthorityWizard
+import uk.gov.hmrc.test.ui.pages.ProfileSetup.{NIPHLNumberPage, NIPHLQuestionPage, NIRMSQuestionPage, UKIMSNumberPage}
+
+class NIPHLNumberStepDef extends BaseStepDef {
+
+  Then("""^(?:I )?(am|should be) on the 'NIPHL reference' page$""") { (amShould: String) =>
+    AuthorityWizard
+      .validLogin()
+    submitPage()
+    UKIMSNumberPage
+      .loadPage()
```

Comment:
In loadPage method we are verifying page title - can we rename it to verifyPageTitle() ? with that can we add new method to verify content of h1? - assertPageContent() - later when we finalise content we will add those here too